### DISCUSSION TITLE
Nice app!  Is it open to contributors?

### DISCUSSION THREAD

**Discussion by ethanstrominger** [rkotze/git-mob-vs-code#55] [Q&A] [answered]
Nice app!  Is it open to contributors?

**Comment by rkotze** [comment, marked as answer]
Hi,
Yes you're welcome to contribute.

Please create an issue if one does not exist already. Fork the repo and make a pull request referencing the issue.

Have a read of the [contributing guide](https://github.com/rkotze/git-mob-vs-code/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/rkotze/git-mob-vs-code/blob/master/CODE_OF_CONDUCT.md)

Thanks.
